sdk/web-pubsub/web-pubsub-client/src/webPubSubClient.ts

```diff
@@ -521,6 +536,24 @@ export class WebPubSubClient {
     return new WebSocketClientFactory();
   }
 
+  private async _trySendSequenceAck(): Promise<void> {
+    if (!this._protocol.isReliableSubProtocol) {
+      return;
+    }
+    const [isUpdated, seqId] = this._sequenceId.tryGetSequenceId();
+    if (isUpdated) {
+      const message: SequenceAckMessage = {
+        kind: "sequenceAck",
+        sequenceId: seqId!,
```

Comment:
Do we need to check if seqId is undefined? The SequenceAckMessage seems to expect not undefined but tryGetSequenceId may get undefined. Maybe check for isUpdated && seqId?